api/Document.json

```diff
@@ -5425,10 +5425,12 @@
               "version_added": "32"
             },
             "safari": {
-              "version_added": null
+              "version_added": "5.1",
```

Comment:
This doesn't match https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullScreen#browser_compatibility, the event can't have been fired before the API was supported.

Reply:
It appears that `api.Element.requestFullScreen` is incorrect, then.  I just launched Safari 5.1 on BrowserStack and ran `document.forms[0].webkitRequestFullScreen`, and the result came up as a function.  I'll submit a PR to correct it.

Reply:
Thanks! That's https://github.com/mdn/browser-compat-data/pull/11162, would you mind folding those changes into that PR for easier review as a group?

Reply:
I've merged the changes from this PR into that one!  This PR is now no longer needed.